pillar/elastic_stack/elasticsearch/logging.sls

```diff
@@ -0,0 +1,56 @@
+#!jinja|yaml|gpg
+elastic_stack:
+  elasticsearch:
+    configuration_settings:
+      cluster.name: mitodl_ops_cluster
+      discovery.zen.minimum_master_nodes: 3
+      discovery.ec2.tag.escluster: operations
+      gateway.recover_after_nodes: 3
+      gateway.expected_nodes: 5
+      gateway.recover_after_time: 5m
+      repositories:
+        s3:
+          bucket: mitx-elasticsearch-backups
+          region: us-east-1
+      discovery:
+        zen.hosts_provider: ec2
+      cloud.node.auto_attributes: true
+      network.host: [_eth0_, _lo_]
+    plugins:
+      - name: discovery-ec2
+        config:
+          aws:
+            region: us-east-1
+      - name: repository-s3
+
+    # There seems to be a bug in
+    # salt.states.elasticsearch.index_template_present, which causes it to fail
+    # without a message, so we're going to update index templates manually until
+    # it's sorted out. This is where you would specify the template:
+    #
+    # index_templates:
+    #   - name: logstash
+    #     definition:
+    #       template: logstash-*
+    #       settings:
+    #         index:
+    #           refresh_interval: 5s
+    #           number_of_shards: 5
+    #           number_of_replicas: 1
+    #       mappings:
+    #         fluentd:
+    #           dynamic_templates:
+    #             - strings:
+    #                 match_mapping_type: string
+    #                 mapping:
+    #                   type: text
+    #                   fields:
+    #                     raw:
+    #                       type: keyword
```

Comment:
So, I know that the `not analyzed` setting allows us to have a `.raw` attribute for searching items that aren't tokenized. What does the `keyword` type provide us with?

Reply:
`keyword` essentially means not analyzed / tokenized; the whole value of that field is considered as one keyword.

They changed their mapping syntax from version <6 to 6. Ran into this on my last job ...

Reply:
I should have mentioned my motivation for changing this was actually loading data into a test ES6 index on my Docker container and running into mapping errors w the previous iteration of the mapping.